controllers/loadtest_controller.go

```diff
@@ -270,12 +271,12 @@ func (r *LoadTestReconciler) Reconcile(ctx context.Context, req ctrl.Request) (c
 		adjustAvailabilityForDefaults := func(defaultPoolKey, defaultPoolName string) bool {
 			if c, ok := missingPods.NodeCountByPool[defaultPoolKey]; ok && c > 0 {
 				if defaultPoolName == "" {
-					log.Error(errNonexistentPool, "default pool is not defined or does not existed in the cluster", "requestedDefaultPool", defaultPoolKey)
+					logger.Error(errNonexistentPool, "default pool is not defined or does not existed in the cluster", "requestedDefaultPool", defaultPoolKey)
```

Comment:
Wondering how would it look like before.

Reply:
I did modify locally, but failed to commit in my last pr. It would give build error.